docs/content/docs/guide/cli.md

```diff
@@ -126,8 +126,8 @@ GitHub application and the secret with all the information needed in the
 
 ### Repository Creation
 
-`tkn pac create repo` -- will create a new Pipelines as Code Repository
-definition, a namespace where the pipelineruns command and configure webhook. It
+`tkn pac create repo` -- Creates a new Pipelines as Code `Repository` custom resource definition,
+a namespace where the pipelineruns execute, configures webhook, and asks for configuring webhook if GithubApp is configured. It
```

Comment:
> if GithubApp is configured.

what does it mean :)

Reply:
I wanted to mention that while executing `tkn pac create repo` if it sees GithubApp is configured then there will be a ask like below
```
👌 A GitHub App is configured for your cluster, Would you like to setup webhook for your repository?
```

So to address that i have added sentence like below i.e

```
`tkn pac create repo` -- Creates a new Pipelines as Code `Repository` custom resource definition,
a namespace where the pipelineruns execute, configures webhook, and asks for configuring webhook if GithubApp is configured.
```

Let me know if this is confusing or any other suggestion

Reply:
How about this?
```
Adds a Git repository to execute pipelinerunss on Git events
```

Reply:
Updated like below